backends/tfhe-cuda-backend/cuda/include/integer/integer_utilities.h

```diff
@@ -5663,6 +5663,98 @@ template <typename Torus> struct unsigned_scalar_div_rem_buffer {
   }
 };
 
+template <typename Torus> struct signed_scalar_div_rem_buffer {
+  int_radix_params params;
+  bool allocate_gpu_memory;
+
+  CudaRadixCiphertextFFI *numerator_ct;
+
+  int_signed_scalar_div_mem<Torus> *signed_div_mem;
+  int_logical_scalar_shift_buffer<Torus> *logical_scalar_shift_mem;
+  int_scalar_mul_buffer<Torus> *scalar_mul_mem;
+  int_sub_and_propagate<Torus> *sub_and_propagate_mem;
+  int_sc_prop_memory<Torus> *scp_mem;
+
+  signed_scalar_div_rem_buffer(
+      cudaStream_t const *streams, uint32_t const *gpu_indexes,
+      uint32_t gpu_count, const int_radix_params params,
+      uint32_t num_radix_blocks, const bool allocate_gpu_memory,
+      const bool anticipated_buffer_drop, uint32_t num_scalar_bits_for_div,
+      uint32_t num_scalar_bits_for_mul, bool is_absolute_divisor_one,
+      bool is_divisor_negative, bool l_exceed_threshold,
+      bool is_absolute_divisor_power_of_two, bool is_divisor_zero,
+      bool multiplier_is_small, uint64_t *size_tracker) {
```

Comment:
Now size_tracker should be passed as `uint64_t & size_tracker` please, as we merged: https://github.com/zama-ai/tfhe-rs/pull/2504